### PR TITLE
report: Add the output of failed tests to the report, with a simple toggle to show/hide

### PR DESCRIPTION
**Pull request by wash-amzn**
This makes it a lot easier to view the reasons for test failures. I would have added the output for all tests (not just failed), but that gets the page to nearly 2MB. Here it's still under 200K.

Github won't let me attach an html file. I'll find another way to get a preview link in here.
Edit: preview generated the same way the daily action does it: https://htmlpreview.github.io/?https://github.com/wash-amzn/arm64-python-wheel-tester/blob/report/index.html

### CODE REVIEW

**Review by geoffreyblake — COMMENTED** [review]
This looks fine to me, but will let a more HTML savvy person sign-off on it.

File: test/process-results.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 from functools import reduce
 from collections import OrderedDict, defaultdict
 from datetime import datetime, timedelta
+from html import escape as html_escape
```
Comment (AWSjswinney) [inline test/process-results.py:11]: I have a slight preference for using `import html` and then calling the function via `html.escape`, but I see that I already used `html` as a variable name. Perhaps a future refactoring could fix that.

**Review by AWSjswinney — APPROVED** [review]
Looks good. Thanks.

File: test/process-results.py (hunk 1 above)
Comment (wash-amzn) [reply test/process-results.py:11]: Yeah, same here, but I didn't want to explode the diff. That module only contains two functions, so I was fine with this.

**Pull request merged**